mifosng-provider/src/main/resources/sql/migrations/core_db/V219__opening_balaces_setup.sql

```diff
@@ -0,0 +1,34 @@
+-- Example: INSERT INTO `acc_gl_account` 
+--	(
+-- `name`, `parent_id`, `hierarchy`, `gl_code`, `disabled`, `manual_journal_entries_allowed`,
+--	`account_usage`, `classification_enum`, `tag_id`, `description`
+--	)
+--	VALUES 
+--	(
+--	'Opening Balances Contra Account', NULL, '.', 'OBCA', 0, 1, 
+--	1, 3, NULL, NULL
+--	);
+
+INSERT INTO `c_configuration` 
+	(
+		`name`, 
+		`value`, 
+		`enabled`
+	)
+	VALUES 
+	(
+		'office-opening-balances-contra-account', 
+		0, -- Or Example: (SELECT id FROM acc_gl_account WHERE gl_code = 'OBCA' ),
+		1);
+
```

Comment:
@binnygopinath and @madhukar547 : acc_gl_financial_activity_account was created specifically for this purpose and the contra account should be defined there